key.core/src/main/java/de/uka/ilkd/key/settings/AbstractPropertiesSettings.java

```diff
@@ -212,9 +213,12 @@ public void parseFrom(String value) {
         @Override
         public void set(T value) {
             T old = get();
-            properties.setProperty(key, toString.apply(value));
-            if (value != null && !value.equals(old)) {
-                fireSettingsChange();
+            // only store non-null values
```

Comment:
Caveat: This means, we cannot use "set(null)" to unset an entry. Perhaps it is not needed since it was not possible till now anyway.

Reply:
`set(null)` wasn't possible prior to the LookAndFeel configuration option (see [that commit](https://github.com/KeYProject/key/commit/d0cdd23161e94ebfc02aef4bba910fa4ef0c0bb6#diff-bbf9e4af8a6ea6c88dd1c0e2fa53264e5e087c21e119ddf27dbda007f00d903cR216))